packages/combo-box/test/overlay-position.test.js

```diff
@@ -213,78 +213,4 @@ describe('overlay position', () => {
       });
     });
   });
-
-  describe('overlay resizing', () => {
-    const minHeight = 116;
-    const inputUnderline = 2;
-
-    it('should resize to bottom of the screen', () => {
-      comboBox.open();
-
-      moveComboBox(xCenter, yBottom - minHeight - inputField.getBoundingClientRect().height - inputUnderline, 300);
-
-      expect(overlayPart.getBoundingClientRect().bottom).to.be.at.most(window.innerHeight);
-    });
-
-    it('should resize to top of the screen', () => {
-      moveComboBox(xCenter, yBottom, 300);
-      comboBox.open();
-
-      moveComboBox(xCenter, minHeight + inputUnderline, 300);
-
-      expect(overlayPart.getBoundingClientRect().top).to.be.at.least(0);
-    });
-  });
-});
-
-describe('fixed position target', () => {
```

Comment:
These tests were added https://github.com/vaadin/vaadin-combo-box/pull/158. Back then, combo-box was using `position: absolute` for the overlay in certain cases. Corresponding logic was removed in #2497. Also, combo-box overlay now uses styles from `vaadin-overlay` that set `position: fixed` so these tests always pass.